.github/actions/next-stats-action/package.json

```
@@ -17,5 +17,10 @@
   },
   "devDependencies": {
     "typescript": "5.1.6"
-  }
+  },
+  "engines": {
+    "node": ">=16.8.0",
+    "pnpm": "8"
```

```suggestion
    "pnpm": "8.6.11"
```

Should we be more specific here in case someone doesn't have corepack enabled?